ietf/utils/management/commands/coverage_changes.py

```diff
@@ -235,7 +235,7 @@ def handle(self, *args, **options):
             # verbosity = int(options.get('verbosity'))
             if not filenames:
                 filenames = [
-                    getattr(settings, 'TEST_COVERAGE_MASTER_FILE'),
+                    getattr(settings, 'TEST_COVERAGE_main_FILE'),
```

Comment:
This isn't what is intended, and I suspect it might break the interface with coverage.